app/Http/Controllers/CheckoutController.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Artwork;
+use Illuminate\Http\Request;
+use Illuminate\Support\Facades\Http;
+
+class CheckoutController extends Controller
+{
+    public function index()
+    {
+        $randomItemsIds = Artwork::all()->random(rand(1, 5))->pluck('id');
+        session(['wishlist.items' => $randomItemsIds]);
+        $wishlistItems = Artwork::whereIn(
+            'id',
+            session('wishlist.items')
+        )->get();
+
```

Comment:
here we should probably let the customer know if any of the artworks he's added to his wishlist are no longer available (filtering by the identifier mentioned in the next comment)